osu.Game.Tests/Visual/Menus/TestSceneToolbarClock.cs

```diff
@@ -76,5 +88,37 @@ public void TestLongGameTime()
         {
             AddStep("Set game time long", () => mainContainer.Clock = new FramedOffsetClock(Clock, false) { Offset = 3600.0 * 24 * 1000 * 98 });
         }
+
+        [Test]
+        public void TestHoverBackground()
+        {
+            Box hoverBackground = null;
+
+            AddStep("Retrieve hover background", () => hoverBackground = (Box)toolbarClock.Children[0]);
```

Comment:
Also testing with index based lookups like this will probably cause more failures on the test than the actual code it's testing. ie. if the children order or content changes, the test is going to fail when the actual logic probably hasn't.

These kinds of test should be avoided.

Reply:
This was the thing I was doubtful about that I mentioned in the Discord chat before opening this PR.
Thanks for clarifying, this was input I really needed.